apps/win32/package.json

```diff
@@ -45,22 +45,23 @@
     "@uifabricshared/build-native": "^0.1.1",
     "@fluentui-react-native/eslint-config-rules": "^0.1.1",
     "@wdio/allure-reporter": "5.22.4",
-    "@wdio/appium-service": "5.18.2",
-    "@wdio/cli": "5.23.0",
-    "@wdio/jasmine-framework": "5.18.6",
-    "@wdio/local-runner": "5.23.0",
-    "@wdio/spec-reporter": "5.22.4",
-    "@wdio/sync": "5.20.1",
     "allure-commandline": "2.13.0",
-    "appium": "1.17.1",
+    "appium": "1.20.0",
     "metro-config": "^0.66.2",
     "metro-react-native-babel-preset": "^0.66.2",
     "react-native-svg-transformer": "^0.14.3",
     "react-test-renderer": "17.0.1",
     "ts-node": "^8.10.1",
     "tsconfig-paths": "^3.9.0",
     "typescript": "3.8.3",
-    "webdriverio": "5.22.4"
+    "webdriverio": "^6.5.0",
+    "appium-mac2-driver": "0.14.1",
```

Comment:
win32 needs the mac2 driver?

Reply:
No, good catch. Removing for Win32/Windows now. Thanks